.github/workflows/_xpu-test.yml

```diff
@@ -42,14 +42,24 @@ on:
         required: false
         type: string
         default: ""
+      use-gha:
+        required: false
+        type: string
+        default: ""
+        description: If set to any value, upload to GHA. Otherwise upload to S3.
+      s3-bucket:
```

Comment:
I think s3-bucker is not really required here 
Since you already have it here:
https://github.com/pytorch/pytorch/blob/main/.github/actions/upload-test-artifacts/action.yml#L17